build_all.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eux
+
+for bp in $(find buildpack -name Dockerfile); do
+	path=$(dirname $bp)
+	flavor=$(basename $path)
+	(cd $path; docker build -t stackstorm/buildpack:$flavor .) || exit -1
+done
+
+for pb in $(find packagingbuild -name Dockerfile); do
+	path=$(dirname $pb)
+	flavor=$(basename $path)
+	(cd $path; docker build -t stackstorm/packagingbuild:$flavor .) || exit -1
+done
+
+for pt in $(find packagingtest -name Dockerfile); do
+	path=$(dirname $pt)
+	flavor=$(basename $path)
+	if [[ "$flavor" == "systemd" ]]; then
+		path=$(dirname $pt)
+		flavor=$(basename $(dirname $path))
```

Comment:
If the change is supposed to be the same as what we currently have then this wil lneed a change.
This will result in the tag centos7, rather than what we have now of centos7-systemd. (see st2-packages/docker-compose.yml).
So you could just change this line to be:
`flavor=$(basename $(dirname $path))-systemd`